R/RSF/build_tarballs.jl

```diff
@@ -0,0 +1,1887 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RSF"
```

Comment:
Why is this called RSF and not Madagascar?

Reply:
The main library is `libdrsf` but can change the name

Reply:
https://docs.binarybuilder.org/stable/building/#Name
> It should be the name of the upstream package, not for example that of a specific library or executable provided by it, even though they may coincide.